__tests__/app.test.js

```diff
@@ -6,16 +6,53 @@ const seed = require("./../db/seeds/seed")
 
 beforeEach(() => seed({ topicData, userData, commentData, articleData }))
 afterAll(() => db.end())
-
-
-//articles // add another request test
+//articles
 describe("GET /api/articles",() => {
+    test("status: 200, responds with an array of article objects, sorted in descending order", () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body: { article }}) => {
+            article.forEach((article) => {
+                expect(article.article_id).toEqual(expect.any(Number));
+                expect(article.topic).toEqual(expect.any(String));
+                expect(article.title).toEqual(expect.any(String));;
+                expect(article.votes).toEqual(expect.any(Number));
+                expect(article.author).toEqual(expect.any(String));
+                expect(article.created_at).toEqual(expect.any(String))
+                expect(article.comment_count).toEqual(expect.any(String))
+                expect(article.article_img_url).toEqual(expect.any(String));
+            })
+            expect(article.length).toBe(13)
+            expect(article.body).toBe(undefined)
```

Comment:
I'd prefer this to be something along the lines of article.hasOwnProperty('body') to be false.